### PR TITLE
russian letters in search

### PR DESCRIPTION
**Pull request by acinonyxjubatus**
regarding https://meta.discourse.org/t/tag-search-in-tags-dropdown-box-does-not-work-for-non-english-characters/53731/4
https://meta.discourse.org/t/multi-language-tags-search/58189?source_topic_id=63279

### CODE REVIEW

File: lib/search.rb [hunk 1]
```diff
@@ -397,7 +397,7 @@ def self.advanced_filters
     end
   end
 
-  advanced_filter(/^\@([a-zA-Z0-9_\-.]+)/) do |posts,match|
+  advanced_filter(/^\@([a-zA-Zа-яА-Я0-9_\-.]+)/) do |posts,match|
```
Comment (SamSaffron) [inline lib/search.rb]: I am a bit confused, usnernames do not allow for russian chars

Comment (acinonyxjubatus) [inline lib/search.rb]: Maybe only for 425 line then.

**Comment by Toxu-ru** [comment]
Is this not the solution for this?
Change only for line 448
https://meta.discourse.org/t/tag-search-in-tags-dropdown-box-does-not-work-for-non-english-characters/53731/4

File: lib/search.rb (hunk 1 above)
Comment (Toxu-ru) [inline lib/search.rb]: only for 425 line.

Comment (Toxu-ru) [inline lib/search.rb]: delete

File: lib/search.rb [hunk 2]
```diff
@@ -422,7 +422,7 @@ def self.advanced_filters
     end
   end
 
-  advanced_filter(/tags?:([a-zA-Z0-9,\-_]+)/) do |posts, match|
+  advanced_filter(/tags?:([a-zA-Zа-яА-Я0-9,\-_]+)/) do |posts, match|
```
Comment (Toxu-ru) [inline lib/search.rb]: add

**Comment by ZogStriP** [comment]
Closing in favor of #4886.

**Pull request closed without merging**